File: A. Beautiful Matrix.java
import java.util.Scanner;
 
public class Main{
 
	public static void main(String[] args) {
int x=0,y=0,move;
        Scanner sc = new Scanner(System.in);
        int[][] arr=new int[5][5];
 
        for (int i = 0; i < 5; i++)
        {
		for (int j = 0; j < 5; j++) {
			arr[i][j]=sc.nextInt();
			if(arr[i][j]==1)
			{
				x=i;
				y=j;
			}
		}
	       }
 
        move=Math.abs(x-2)+Math.abs(y-2);
           System.out.println(move);
			sc.close();
	}
 
}

File: A. Beautiful Matrix.c
#include <iostream>
#include <cmath>
 
using namespace std;
int n,arr[5][5];
int main()
{
   for(int i=0;i<5;++i){
       for(int j=0;j<5;++j){
           cin>> arr[i][j];
        if(arr[i][j]==1){
            n= abs(j-2)+abs(i-2);
         cout <<n;
 
       }
 
       }
   }
  return 0;
}